docs/readthedocs/source/doc/Nano/Overview/troubshooting.md

```diff
@@ -0,0 +1,57 @@
+# Troubleshooting Guide for BigDL-Nano
+
+Refer to this section for common issues faced while using BigDL-Nano.
+
+## Installation
+### Why I fail to install openvino==2022.2 when ``pip install bgdl-nano[inference]``?
+
+Please check your system first as openvino 2022.x does not support centos anymore. Refer [OpenVINO release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes-2021.html) for more details.
+
+## Inference
+
+### ``could not create a primitive descriptor iterator`` when using bf16 related methods
+
+Please make sure you use context manager provided by ``InferenceOptimizer.get_context``, you can refer this [howto guide for context manager]() for more details.
```

Comment:
no link to 'howto guide for context manager' ? Maybe we should add that howto guide before merging this PR

Reply:
sure, will merge that PR first.